pkg/oc/cli/cmd/rsync/forwarder.go

```diff
@@ -33,10 +34,9 @@ func (f *portForwarder) ForwardPorts(ports []string, stopChan <-chan struct{}) e
 		Name(f.PodName).
 		SubResource("portforward")
 
-	dialer, err := remotecommand.NewExecutor(f.Config, "POST", req.URL())
-	if err != nil {
-		return err
-	}
+	transport, upgrader, err := spdy.RoundTripperFor(f.Config)
+	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, "POST", req.URL())
```

Comment:
@openshift/cli-review @smarterclayton this moved in upstream and I want to be sure this is all right... can you double check pls?